### PR TITLE
refactor(TopNavbar): remove ForwardArrow & update BackArrow to return to base route

### LINKED ISSUE #607: refactor(NavTopbar): remove forward arrow & back arrow should take you to the base url of that view
Eg, if we are on the collections page & then click into a dataset, the "back" arrow should return us to the collections page.

If we are in the network tab & we click into a dataset, the "back" arrow should return us to the network page.

### PR DESCRIPTION
**Pull request by uhLeeshUh**
closes #607 

- removes ForwardArrow component completely
- updates BackArrow logic to always return to base route. If at base route, then arrow is disabled

### CODE REVIEW

**Review by ramfox — APPROVED** [review]
LGTM!!!!

File: app/components/nav/NavTopbar.tsx [hunk 1]
```diff
@@ -42,12 +41,21 @@ export const NavTopbar: React.FunctionComponent<NavTopbarProps> = ({ title, loca
...
+    return history.push(`/${baseRoute}`)
+  }
+
   return (
     <div className='page-navbar'>
       <div className='row'>
         <div className='nav-buttons'>
-          <a className='back' onClick={() => history.goBack()}><BackArrow /></a>
-          <a className='forward' onClick={() => history.goForward()}><ForwardArrow /></a>
+          {!isBaseRoute && <a className='back' onClick={onBackClick}><BackArrow /></a>}
```
Comment (b5) [inline app/components/nav/NavTopbar.tsx:58]: I've also adjusted the back arrow to not appear at all whenever we're at a base route. keeping a convention established with mobile apps.

**Review by b5 — APPROVED** [review]
made some quick adjustments. LGTM!

File: app/components/nav/NavTopbar.tsx (hunk 1 above)
Comment (uhLeeshUh) [reply app/components/nav/NavTopbar.tsx:58]: sounds great, thanks!

File: app/components/nav/NavTopbar.tsx [hunk 2]
```diff
@@ -42,12 +41,21 @@ export const NavTopbar: React.FunctionComponent<NavTopbarProps> = ({ title, loca
     setModal({ type: ModalType.ExportDataset })
   }
 
+  // determines if route is at base route (e.g. /collection or /network)
+  const isBaseRoute = Object.keys(match.params).length === 0
```
[the remark this answers is not in the document]
Comment (uhLeeshUh) [reply app/components/nav/NavTopbar.tsx:45]: No prob. Sounds good to me

**Pull request merged**